db/migrations/20260817153015_add_report_fields/up.sql

```
@@ -0,0 +1,32 @@
+        'COMMUNICATION',
+        'GOOD_EXPERIENCE',
+        'OTHER'
+      )
+    ),
+  add constraint "Post_rating_check"
+    check ("rating" is null or "rating" between 1 and 5),
+  add constraint "Post_status_check"
+    check ("status" in ('PUBLISHED', 'HIDDEN'));
+
+create index "Post_public_feed_idx"
+  on "Post" ("status", "createdAt" desc, "id" desc);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)